test/assertions_test.rb

```diff
@@ -0,0 +1,70 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class AssertionsTest < ActionController::TestCase
+      class MyController < ActionController::Base
+        def render_using_serializer
```

Comment:
I followed the same Idea of the [serialization_test.](https://github.com/rails-api/active_model_serializers/blob/master/test/action_controller/serialization_test.rb). In fact the code I am testing here is in the `lib/active_model/serializer/assertions.rb` not sure if this goes in the `test` folder or in the `action_controller` I can move if you prefer :smile: